### PR TITLE
Feature/frontend error handling

### PR DESCRIPTION
**Pull request by hritik5102**
### Description

- Add toaster component on layouts
- Handle change password behavior
- Handle register failure behavior
- Handle logout user failure behavior
- Handle follow and unfollow user failure behavior
- Handle tweet action failure behavior
- Handle single tweet failure behavior
- Handle Compose tweet failure behavior
- Handle account detail failure behavior
- Handle account detail success behavior
- Handle change password success behavior

### CODE REVIEW

File: frontend/src/constants/ui/strings.js [hunk 1]
```diff
@@ -71,6 +71,7 @@ export default {
     },
     ERROR: {
       userNotFound: "User not found",
+      somethingWentWrong: "Oops, Something went wrong. Please try again later",
```
Comment (Saunved) [inline frontend/src/constants/ui/strings.js]: This is not a good error message. I know that we are used to seeing such messages, but they don't offer any useful information to the end user.

Change it to this:
"Sorry, we were unable to process your request. You can try again later or report this issue to bugs@hs-woofer.com".

File: frontend/src/pages/_app.js [hunk 2]
```diff
@@ -21,19 +21,29 @@ export default function App({ Component, pageProps }) {
...
+          <BlankLayout children={page} />
+          <Toaster position="top-center" reverseOrder={false} />
+        </>
+      );
     }
 
     if (pageIsForSettings) {
       return (
-        <Layout>
+        <Layout children={page}>
```
Comment (Saunved) [inline frontend/src/pages/_app.js:34]: I don't think you need this.

Imo it should be a fragment with the nested SettingsLayout serving the actual children.

Please make this change and test it out once.

File: frontend/src/pages/auth/register.jsx [hunk 3]
```diff
@@ -2,10 +2,8 @@ import strings from "@/constants/ui/strings";
 import { Dog } from "phosphor-react";
 import { useState, useEffect } from "react";
 import Link from "next/link";
-import Input from "@/components/Form/Input";
-import PasswordLabel from "@/components/Form/PasswordLabel";
 import { registerRoute } from "@/constants/routes";
-import { toast } from "react-hot-toast";
+import toast from "react-hot-toast";
```
Comment (Saunved) [inline frontend/src/pages/auth/register.jsx:6]: Shouldn't this be { toast } ?

**Review by Saunved — CHANGES_REQUESTED** [review]

File: frontend/src/constants/ui/strings.js (hunk 1 above)
Comment (hs-hritik) [reply frontend/src/constants/ui/strings.js]: Ack

**Review by Saunved — APPROVED** [review]
Approved assuming the comments are fixed

**Review by nrkin — APPROVED** [review]

**Pull request merged**